pymc4/tests/test_random_variables.py

```diff
@@ -0,0 +1,97 @@
+"""
+Tests for PyMC4 random variables
+"""
+import pytest
+import numpy as np
+
+
+from .. import _random_variables
+
+
+@pytest.fixture(scope="session")
+def tf_supported_args():
+    """Provide arugments for each supported Tensorflow distribution"""
+    _tfp_supported_args = {
+        "Bernoulli": {"probs": 0.5},
```

Comment:
why not directly map the RV classes to their objects? Saves the `getattr` lookup below.

Reply:
This sounds way better. Will do

Reply:
I remember why I didn't do this.

I wanted a couple things
1. An ordered object of distribution strings so I can pass them to the id argument function
2. A mapping of parameters to distributions
3. A guarantee that when testing a method, the id string matches the distribution being tested
4. Only one place where order is defined

If I mapped directly from rv.classes to their parameters the tests would still work, but I wouldn't have a way to get the right distribution name string to the id argument.

The other reason I felt this might be better is that when testing the distribution name is visible, even in pdb. If I mapped RV Classes, in a debugger they would show up as memory addresses since we haven't defined a `__repr__` for `RandomVariable` yet

There also is the possibility I'm wrong or explaining things poorly so let me know if this doesn't make sense.

Reply:
I don't follow that reasoning. 
> An ordered object of distribution strings so I can pass them to the id argument function
This isn't ordered once you created it as it's a dict. You could sort it but I don't see why you would want to do that.
> A mapping of parameters to distributions
This is a mapping of parameters to strings of distributions.
> A guarantee that when testing a method, the id string matches the distribution being tested
I don't see the benefit of that. You only increase the chance of having typos in the strings.
> Only one place where order is defined. If order is defined in two places, then coordination of order is human reliant, which means it'll definitely get messed up.
Where is order defined here in the first place? And why would that be important?

> If I mapped directly from rv.classes to their parameters the tests would still work, but I wouldn't have a way to get the right distribution name string to the id argument.
That's true, but do we need that?
> The other reason I felt this might be better is that when testing the distribution name is visible, even in pdb.
We do have the class name that will get printed.

In sum, I don't really see the benefit, other than easier lookup when trying to get parameters manually but I don't see the use-case as it's just a test. But I do think it's more brittle to have that level of indirection.

Reply:
OK, that looks way nicer. I wonder if we should just change the `__repr__` instead though.

Reply:
I think I reformulated this to meet the ask. Let me know what you think!